pkg/controller/controller.go

```diff
@@ -116,14 +122,25 @@ func New() *Controller {
 			if err != nil {
 				return nil, err
 			}
-
 			return handler(ctx, req)
 		},
 	)
 
 	csi.RegisterIdentityServer(controller.Server, controller)
 	csi.RegisterControllerServer(controller.Server, controller)
 
+	sigc := make(chan os.Signal, 1)
+	signal.Notify(sigc,
```

Comment:
May consider adding more signals to cover. A SIGHUP, SIGINT, or SIGTERM signal causes the program to exit. A SIGQUIT, SIGILL, SIGTRAP, SIGABRT, SIGSTKFLT, SIGEMT, or SIGSYS signal causes the program to exit with a stack dump.

Reply:
Okay, sounds good.